src/settings.c

```diff
@@ -169,6 +169,27 @@ void load_settings(char *cmdline_config_path)
                 "Truncating long lines or do word wrap"
         );
 
+        {
+                char *c = option_get_string(
+                        "global",
+                        "ellipsize", "-ellipsize", "",
```

Comment:
Since we know middle is the default it's a good idea to set this here else the fact that middle is used by default is only because it's the first entry in the enum.

Reply:
Same here, I based my code on the alignment, that why I did not put `"middle"` as the default value.
I'll put back the enum in a logical order too (as `middle` doesn't have to be the first one anymore).
Does that mean an update should also be done on the alignment to keep some consitency (that also include the handle of the `default` case)?

Reply:
> Does that mean an update should also be done on the alignment to keep some consitency (that also include the handle of the default case)?

Yes but it's out of the scope of this PR (let's keep things organized). Unfortunately there are a lot of such inconsistencies in the code, I keep putting off doing a general minor cleanup with such changes.

Reply:
> So I don't know how you want me to handle this case.
Should I throw an error saying something went wrong and exiting?

An `assert` statement like should be all that's needed. It's not expected for the user to ever see that error so there's no need to be fancy with log messages and since there is specific error check here, `assert(false)` should suffice.

Reply:
I needed to set the ellipsize value in case the user value is invalid, to fallback on the default behaviour (as `middle` is not the first anymore).